### PR TITLE
feat: Add tests DatabaseHelperTest 

### PR DESCRIPTION
**Pull request by KarpovIvan**
## Description

task: [24657](https://github.com/camunda/camunda/issues/24657)

## Checklist

- [ ] for CI changes:
  - [ ] structural/foundational changes signed off by [CI DRI](https://github.com/cmur2)
  - [ ] [ci.yml](https://github.com/camunda/camunda/blob/main/.github/workflows/ci.yml) modifications comply with ["Unified CI" requirements](https://github.com/camunda/camunda/wiki/CI-&-Automation#workflow-inclusion-criteria)

## Related issues

closes #

### CODE REVIEW

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import io.camunda.optimize.dto.optimize.datasource.ZeebeDataSourceDto;
+import org.junit.jupiter.api.Test;
+
+public class DatabaseHelperTest {
+
+  public static final String ALIAS = "alias";
+
+  @Test
+  void shouldConstructKeyWithDatabaseTypeAndEngineAlias() {
+    // Given
```
Comment (RomanJRW) [inline optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java:24]:
```suggestion
    // given
```
Nitpick, but please do this everywhere to align with convention

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    final String databaseType = DataImportSourceType.ZEEBE.toString();
+
+    // When
```
Comment (RomanJRW) [inline optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java:27]:
```suggestion
    // when
```
Nitpick, but please do this everywhere to align with convention

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    final String result = DatabaseHelper.constructKey(databaseType, ALIAS);
+
+    // Then
```
Comment (RomanJRW) [inline optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java:30]:
```suggestion
    // then
```
Nitpick, but please do this everywhere to align with convention

**Review by RomanJRW — APPROVED** [review]
LGTM, just some minor comments on the convention 👍

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java (hunk 1 above)
Comment (KarpovIvan) [reply optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java]: fixed

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java (hunk 2 above)
Comment (KarpovIvan) [reply optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java]: fixed

File: optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java (hunk 3 above)
Comment (KarpovIvan) [reply optimize/backend/src/test/java/io/camunda/optimize/service/util/DatabaseHelperTest.java]: fixed

**Pull request merged**